pkg/systemd/quadlet/quadlet.go

```diff
@@ -958,7 +958,7 @@ func defaultOneshotServiceGroup(service *parser.UnitFile, remainAfterExit bool)
 // The original Network group is kept around as X-Network.
 // Also returns the canonical network name, either auto-generated or user-defined via the
 // NetworkName key-value.
-func ConvertNetwork(network *parser.UnitFile, name string, unitsInfoMap map[string]*UnitInfo, isUser bool) (*parser.UnitFile, error, error) {
+func ConvertNetwork(network *parser.UnitFile, _ string, unitsInfoMap map[string]*UnitInfo, isUser bool) (*parser.UnitFile, error, error) {
```

Comment:
If we don't need that arg let's just remove it and simplify the callers. We only need to use `_` if we need the arg to satisfy an interface or some other function signature or need to keep API compatibility which should not be the case here.